src/main/resources/static/messages/messages_nl.js

```diff
@@ -93,7 +93,8 @@
         'popover: Laboratory techniques': 'Indien van toepassing: welke labtechnieken (bijv. kiembaanmutatieanalyse/moleculaire technieken, immuunhistochemie, tissue micro array etc.) worden gebruikt?',
         'Request type': 'Benodigde data en/of materialen',
         'popover: Request type': 'Benodigde data en/of materialen. Voor uitleg over PALGA-gegevens, bijv. welke gegevens een excerpt bevat, zie <a target="_blank" tabindex="-1" href="http://www.palga.nl/gegevensaanvragen/zoekvragen.html">www.palga.nl/gegevensaanvragen/zoekvragen.html</a>.',
-
+        'No informed consent form uploaded.': 'Geen informed consent form geupload',
```

Comment:
Wel of geen punt?

Reply:
form = formulier

Reply:
Volgens onze taal (https://onzetaal.nl/taaladvies/engelse-werkwoorden-met-een-u-nederlandse-vervoeging/) is het geüpload dus dat is denk het handigst